command/command.ts

```diff
@@ -953,7 +959,9 @@ export class Command<
           prepend: true,
           action: function () {
             this.showVersion();
-            Deno.exit(0);
+            if (this.exitOnHelp) {
+              Deno.exit(0);
+            }
```

Comment:
And here we should call the function.
```suggestion
            if (this.shouldExitOnHelp()) {
              Deno.exit(0);
            }
```
The function could look something like this.
```ts
private shouldExitOnHelp(): boolean {
  return this.exitOnHelp ?? !!this._parent?.shouldExitOnHelp();
}
```

Reply:
For shouldExitOnHelp.

With using `??` the the variable should probably be switched to be undefined by default else it wouuld never pickup the parents value.

Using `||` would result in any subcommand being unable to not exit if the parent is true,

Using `&&` would result in being unable to overide a parents  `false`;

The alternative would be to change the flag from `exitOnHelp` to `dontExitOnHelp` default it to `false` then it can be treated just like throwOnError with `||`.

Reply:
yeah sry it was just a quick example that may not work, not tested. but i think it should be possible to override the behavior of the child command from the parent command same like with the `throwErors()` method. so just do it how you think it can work 😉